src/main/java/gov/cabinetoffice/api/config/S3ConfigProperties.java

```diff
@@ -13,7 +13,7 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @Configuration("s3ConfigurationProperties")
-@ConfigurationProperties(prefix = "aws")
+@ConfigurationProperties(prefix = "s3")
```

Comment:
if we don't change the application properties file to match this then the application will break

Reply:
Will revert this change.